packages/cli/src/databases/utils/migrationHelpers.ts

```diff
@@ -39,30 +38,41 @@ export function loadSurveyFromDisk(): string | null {
 	}
 }
 
-let logFinishTimeout: NodeJS.Timeout;
+let runningMigrations = false;
 
-export function logMigrationStart(migrationName: string, disableLogging = inTest): void {
-	if (disableLogging) return;
+function logMigrationStart(migrationName: string): void {
+	if (inTest) return;
 
-	if (!logFinishTimeout) {
+	if (!runningMigrations) {
 		logger.warn('Migrations in progress, please do NOT stop the process.');
+		runningMigrations = true;
 	}
 
 	logger.debug(`Starting migration ${migrationName}`);
-
-	clearTimeout(logFinishTimeout);
 }
 
-export function logMigrationEnd(migrationName: string, disableLogging = inTest): void {
-	if (disableLogging) return;
+function logMigrationEnd(migrationName: string): void {
+	if (inTest) return;
```

Comment:
I think having the messages is always useful, especially if running when testing, don't you think?

Reply:
the logic hasn't really changed. we were never passing in a second argument, and `disableLogging` was always `inTest`.